environments/dev-env/CONTRIBUTING.md

```diff
@@ -0,0 +1,510 @@
+# KCP-Edge Contributor Development Environment
+
+## Required Packages:
+   - ko: https://ko.build/install/ 
+   - gcc
+   - docker 
+   - jq
+   - make
+   - go (version expected 1.19)
+   - kind
+   - kubectl  
```

Comment:
Shouldn't we say something about version here, for consistency with the fact that kcp is based on Kubernetes version 1.24.3?

I think that exact match on version is not required; ideally we would document here the range of usable versions for each.

Reply:
@MikeSpreitzer +/- 0.1 in the version is allowed. I had to force install kubectl v1.25.3 and not latest

Reply:
What sort of force are you talking about?  Installing kubectl is its own step, I am not aware of any force required to get whatever version you choose.